src/build/optimize-streams.ts

```diff
@@ -116,9 +137,13 @@ class JSBabelTransform extends GenericOptimizeTransform {
  */
 export class JSDefaultCompileTransform extends JSBabelTransform {
   constructor() {
-    super({
+    super('babel-compile', {
       presets: [babelPresetES2015NoModules],
-      plugins: [externalHelpersPlugin],
+      plugins: [
+        externalHelpersPlugin,
+        babelObjectRestSpreadPlugin,
```

Comment:
did you look into babel-preset-env?

Reply:
just read about it. seems like we should switch but haven't tried it out yet. actually the preset plugins now warn you to move to it.